src/api/flaskr/service/shifu/dtos.py

```
@@ -220,499 +211,409 @@ def __json__(self):
+class OutlineEditDto:
+    outline_id: str
+    outline_no: str
+    outline_name: str
+    outline_desc: str
+    outline_type: int
+    outline_level: int
 
-    def __init__(self, prompt: str = None, variables: list[str] = None):
-        self.prompt = prompt
-        self.variables = variables if variables is not None else []
+    def __init__(
```

Pylint (3.3.7)

[refactor] Too many arguments (6/5)

(R0913)